x/auth/client/rest/rest.go

```diff
@@ -12,12 +12,6 @@ func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, storeName string)
 		"/auth/accounts/{address}",
 		QueryAccountRequestHandlerFn(storeName, context.GetAccountDecoder(cliCtx.Codec), cliCtx),
 	).Methods("GET")
-
-	// TODO: Change path or mount under x/bank if possible
-	r.HandleFunc(
```

Comment:
I would prefer to keep it under the @deprecated category and remove it after a release after it has been deprecated.
I've talked with a lot of people this weekend that would love to have time to adapt to our changes in their clients/apps.

Reply:
and voluntarily keep duplicated but explicitely deprecated code, so it's easy to find and to kill before the next release, let's say first PR after a release may the flushing all the deprecated stuff

Reply:
Yeah we could do this - we should use `// DEPRECIATED will be destroyed after release 0.37.0` or something like that

Reply:
The plan then would become to remove `/bank/balances` altogether because `/auth/accounts` already provides account balances, wouldn't it? I'd agree to that

Reply:
This is superfluous. We're including the changes in this PR in a breaking release anyway. I see now need to keep this around.

Reply:
We still need this endpoint. Removing it would require clients to parse accounts which can vary in structure (vesting vs non-vesting) and that would make it a huge PIA for them.